x-pack/plugins/ml/public/explorer/explorer_charts/legacy_utils.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function getChartContainerWidth() {
```

Comment:
I think this needs a comment as to why it is `legacy_utils` and what the plan for it is

Reply:
Done in 17e2f4b68a.